src/foundations/shape/Shape.props.ts

```diff
@@ -26,6 +26,11 @@ type PShape = {
      * @default 'div'
      * */
     component?: TShapeElement;
+    /**
+     * If the component is an input, which type
+     * @default 'text'
```

Comment:
default values should always be present in the union.
This would need to be extracted to `Text.types.ts` for consistency (possibly extend/use from available HTML types?)

Reply:
Sorry I am not sure I understand this comment. Do you mean this should be a type of the Text component?

Reply:
Oh no ... I am sorry ... of course it should be extracted to `Shape.types.ts` ... i might have been influenced by the `@default 'text'` here! :D

Reply:
Ooooh :D